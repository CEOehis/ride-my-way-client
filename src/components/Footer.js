import React from 'react';

const Footer = () => (
  <footer className="app-grid orange-bg">
    <div className="layout-grid">
      <p>Â© 2018. Hand crafted by Celestine Ekoh-Ordan</p>
    </div>
  </footer>
);

export default Footer;
