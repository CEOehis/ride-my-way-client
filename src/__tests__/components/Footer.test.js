import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../../components/Footer';

describe('Form submit footer component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Footer submitting />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
