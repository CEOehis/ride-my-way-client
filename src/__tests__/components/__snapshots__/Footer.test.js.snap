// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form submit footer component should render correctly 1`] = `
<footer
  className="app-grid orange-bg"
>
  <div
    className="layout-grid"
  >
    <p>
      Â© 2018. Hand crafted by Celestine Ekoh-Ordan
    </p>
  </div>
</footer>
`;
